Automattic/pocket-casts-android
# Deselect Chapters - Disable for user files

Part of: #1807 

## Description
Due to syncing not being available for user files on v1, we won't enable deselecting chapters on user files.

Corresponding iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/1498

Podcast with chapters you can use to test:

* Upgrade: https://pca.st/upgrade

## To test

Download an episode with chapters and save it on your computer.

1. Drag and drop the file to your simulator
2. Add it to Profile -> Files
3. Play it
4. Go to the Chapters tab
5. ✅ You should not see the "Skip Chapters" header
6. Play a "podcast episode" that has chapters
7. Go to Chapters tab
8. ✅ You should see the "Skip Chapters" header

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/55178cbb-2174-43f6-ac27-cb69985f3479



## Checklist
 
#### I have tested any UI changes...
